src/Features/LanguageServer/Protocol/Features/TaskList/TaskListListener.cs

```diff
@@ -1,156 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Removed this indirection entirely.  There was no need for it.

Reply:
No need for it, unless we ship Roslyn inside IDEs other than Visual Studio, like Visual Studio for Mac 😛